client/fresheyes-bot/src/util.ts

```diff
@@ -70,10 +70,12 @@ export function generateIssueBody<T extends Array<Record<string, any>>>(arg: T)
   const comments = arg.length;
   const commentText = comments === 1 ? "comment" : "comments";
   const reviewersText = authors === 1 ? "reviewer" : "reviewers";
+  const isBitcoinBot = arg.find((item) => item?.user?.login.toLowerCase() === "DrahtBot".toLowerCase());
+  const botComment = isBitcoinBot !== undefined ? "1 bot" : "";
```

Comment:
Let's add a comma so the sentence is structured grammatically 

```suggestion
  const botComment = isBitcoinBot !== undefined ? ", 1 bot" : "";
```

Reply:
Alright

Reply:
There's still an extra space after reviewers.
I'm beginning to think we should use `and` instead of a comma.
```js
const botComment = isBitcoinBot !== undefined ? "and 1 bot" : "";

  return [
    {
      body: `There were ${comments} ${commentText} left by ${authors} ${reviewersText} ${botComment} for the pull request`,
      created_at: arg[0].created_at,
      key: "issue",
    },
  ];
  ```